tests/Fixtures/e2e/Codeception_Basic/run_test.bash

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+if [[ $PHPDBG = 1 ]]
```

Comment:
@borNfreee told that he had an issue with this variant on his mac. Could you please change this line to the `if [ "$PHPDBG" = "1" ]` ?

Reply:
I had an issue with if [[ $PHPDBG=1 ]] on my Mac.  if [[ $PHPDBG = 1 ]] works for me.

@borNfreee Does it also work for you?

Reply:
yes, it does not work, see https://github.com/infection/infection/pull/151#discussion_r166079227